content/docs/querying/operators.md

```diff
@@ -17,7 +23,7 @@ The following binary arithmetic operators exist in Prometheus:
 Binary arithmetic operators are defined between scalar/scalar, vector/scalar,
 and vector/vector value pairs.
 
-**Between two scalars**, the behavior is obvious: they evaluate to another
+**Between two scalars**, the behaviour is obvious: they evaluate to another
```

Comment:
Thanks! Yeah, I think for dashes we'll want to stick with a normal "-" (minus, but surrounded by spaces if it's not a dash joining two words) just for simplicity. It's on everyone's keyboard, and is the most common in docs websites, I think.